package.json

```diff
@@ -24,42 +25,7 @@
     "xregexp": "^2.0.0",
     "tmp": "latest"
   },
-  "coffeelintConfig": {
```

Comment:
This really shouldn't have been removed until the specs were converted to pure JS as well.